examples/example-checkbox-header-row.html

```diff
@@ -186,10 +211,11 @@ <h3>Selected Rows:</h3>
     grid = new Slick.Grid("#myGrid", dataView, columns, options);
     grid.setSelectionModel(new Slick.RowSelectionModel({selectActiveRow: false}));
     grid.registerPlugin(checkboxSelector);
+    dataView.syncGridSelection(grid, false, true);
```

Comment:
had to switch 2nd argument to `false` (which is `preserveHidden`) for our use case to work, when set to `true` then our use case doesn't work. I don't know if this is intended